providers/indexing.csv

```diff
@@ -1,10 +1,32 @@
-subquery-free,SubQuery,,Decentralized Data Indexing,Free,FALSE,0,0,FALSE,"[""GraphQL"",""SQL""]","[""Rate limits apply""]","[""Decentralized storage"",""IP Protection""]","[""Data lake logs"",""Usage analytics""]","[""Global""]","[""ETL query stack"",""Decentralized indexing"",""Discord support""]",NULL,NULL,NULL,,,,,
-subquery-pay-as-you-go,SubQuery,,Decentralized Data Indexing,Pay-as-you-go,FALSE,,,FALSE,"[""GraphQL"",""SQL""]","[""Rate limits may apply""]","[""Decentralized storage"",""IP Protection""]","[""Data lake logs"",""Usage analytics""]","[""Global""]","[""ETL query stack"",""Decentralized indexing"",""Hard SLA with compensations""]",NULL,NULL,NULL,,,,,
-subquery-enterprise,SubQuery,,Decentralized Data Indexing,Enterprise,FALSE,,,FALSE,"[""GraphQL"",""SQL""]",,"[""Decentralized storage"",""IP Protection""]","[""Data lake logs"",""Usage analytics""]","[""Global""]","[""ETL query stack"",""Decentralized indexing"",""Hard SLA with compensations""]",NULL,NULL,NULL,,,,,
+alchemy-free,Alchemy,null,Subgraph Indexing,Free,FALSE,$0.00,$0.00,FALSE,"[""GraphQL"",""REST API""]","[""2 subgraph deployments"",""200k queries"",""throughput 5 queries/s"",""stored entities 100K""]","[""Secure API keys"",""Encrypted endpoints""]","[""Real-time logging & usage stats""]","[""Global""]","[""Discord support""]",NULL,NULL,NULL,,,,,
+alchemy-pay-as-you-go,Alchemy,null,Subgraph Indexing,Pay-as-you-go,FALSE,$5/$45/$215/$415/$815/$2k,$20/1M queries,FALSE,"[""GraphQL"",""REST API""]","[""30 subgraph deployments"",""$20/1M queries"",""throughput 30 queries/s"",""stored entities $25/M""]","[""Secure API keys"",""Encrypted endpoints""]","[""Real-time logging & usage stats""]","[""Global""]","[""Priority support""]",NULL,NULL,NULL,,,,,
+alchemy-enterprise,Alchemy,null,Subgraph Indexing,Enterprise,FALSE,Custom,Custom,FALSE,"[""GraphQL"",""REST API""]",,"[""Secure API keys"",""Encrypted endpoints""]","[""Real-time logging & usage stats""]","[""Global""]","[""Unlimited throughput"",""Unlimited apps & webhooks"",""<2m response time VIP support"",""SLA""]",NULL,NULL,NULL,,,,,
+chain-love-free,Chain.Love,0x0fd8fd1dc8162148cb9413062fe6c6b144335dbf,The Graph,Free,FALSE,$0.00,$0.00,FALSE,"[""GraphQL""]",,"[""DDoS Protection"",""IP Protection"",""Domain name protection"",""Custom rate limiting""]","[""Subgraph logs"",""Usage logs""]","[""Japan""]",,NULL,NULL,NULL,,,,,
+chainstack-free,Chainstack,null,The Graph,Free,FALSE,$0.00,null,FALSE,"[""GraphQL""]","[""3M request units per month, 20 request units per subgraph request"",""25 RPS"",""1 subgraph"",""Extra usage is $20 per 1M request units""]","[""DDoS Protection"",""IP Protection""]","[""Subgraph logs"",""Usage analytics""]","[""Global""]","[""Custom Subgraphs"",""GraphQL Queries"",""Standard Support""]",NULL,NULL,NULL,,,,,
```

Comment:
Subgraph indexing and The Graph are the same technologies. Please deduplicate them